include/zephyr/posix/pthread.h

```diff
@@ -39,6 +39,12 @@ extern "C" {
 #define PTHREAD_CANCEL_DEFERRED     0
 #define PTHREAD_CANCEL_ASYNCHRONOUS 1
 
+/* Pthread scope */
+#undef PTHREAD_SCOPE_PROCESS
```

Comment:
it would be ideal if there were no `#undef` here and 2 lines below, but given that this does not really do anything in Zephyr atm, it's not blocking. If the `#undef` is needed to prevent a warning, I suppose that it's ok.

Reply:
I have done that to prevent a warning, if it's a problem I can check on another way.